content/post/2019-04-11-r-pkg-devel.md

```diff
@@ -0,0 +1,681 @@
+---
+title: "How to get help with R package development? R-package-devel and beyond"
+date: '2019-04-01'
+slug: r-package-devel
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+No matter how good your docs reading and search engine querying skills
+are, sometimes as an R package developer you’ll need to ask questions to
+your peers. Where to find them? R-hub has its own
+[feedback](https://github.com/r-hub/rhub) and
+[discussion](https://gitter.im/r-hub/community) venues, but what about R
+package development in general? In this blog post, we’ll have a look at
+the oldest channel for R package development help, the [R-package-devel
```

Comment:
Well, not really the oldest, there was R-devel, and StackOverflow, etc. before.

Reply:
oldest _specific_ then?

Reply:
ok